nixpkgs_review/nix/review-shell.nix

```diff
@@ -26,11 +26,15 @@ let
     in
     map (attrString: lib.attrByPath (lib.splitString "." attrString) null system-pkg) system-attrs;
   attrs = lib.flatten (lib.mapAttrsToList extractPackagesForSystem (import attrs-path));
-  env = local-pkgs.buildEnv {
-    name = "env";
-    paths = attrs;
-    ignoreCollisions = true;
-  };
+  env =
+    local-pkgs.buildEnv {
```

Comment:
I think this needs parens around the attrset update, so `ignoreSingleFileOutputs` makes it into `buildEnv`.

It does seem to work okay once that's changed.